app/javascript/components/ProductEdit/ContentTab/FileEmbed.tsx

```diff
@@ -653,8 +653,8 @@ const FileEmbedNodeView = ({ node, editor, getPos, updateAttributes }: NodeViewP
         ) : null}
       </div>
       {isDropZone ? (
-        <div className="backdrop">
-          <div className="button primary">Create folder with 2 items</div>
+        <div className="absolute inset-0 bg-black/80">
```

Comment:
Should we define a shared variable for backdrops? I feel like we have quite a lot of these at this point (and they're not all the same)

Maybe even a mini-component